e2e/cypress/integration/account_settings/general/nickname_spec.js

```diff
@@ -109,14 +114,90 @@ describe('Account Settings > Sidebar > General', () => {
         cy.get('#teamMembersModal').should('be.visible').within(() => cy.get('.close').click());
     });
 
-    it('Clear the nickname', () => {
+    it('MM-T2060 Nickname and username styles', () => {
+        cy.apiAdminLogin();
```

Comment:
Test failed on me. Please don't use sysadmin as much as possible.
![Screen Shot 2020-12-18 at 6 22 15 PM](https://user-images.githubusercontent.com/5334504/102603876-64466b80-415e-11eb-9fbd-e48c5731958f.png)
![Screen Shot 2020-12-18 at 6 22 27 PM](https://user-images.githubusercontent.com/5334504/102603895-6b6d7980-415e-11eb-8fc7-5d1d063f8d60.png)

```suggestion
```

Reply:
Thanks, I will remove the sysadmin account. But the reason why it might be failing for you is if you don't have DM
s in the list and then the more... button doesn't appear. Perhaps I should create a DM in the test case?

Reply:
For some reason, my sysadmin is using the legacy sidebar.
![Screen Shot 2020-12-19 at 2 52 27 AM](https://user-images.githubusercontent.com/5334504/102650123-59fb9000-41a5-11eb-9785-96e9ff2a0a15.png)